src/Orchard.Specs/Hosting/Orchard.Web/Global.asax.cs

```diff
@@ -29,6 +31,7 @@ public class MvcApplication : HttpApplication {
 
         protected void Application_BeginRequest() {
             Context.Items["originalHttpContext"] = Context;
+            HttpContext.Current.Response.AddHeader("X-Frame-Options", "SAMEORIGIN");
```

Comment:
These two additions are a workaround for the MVC upgrade to 5.2.7 (the breaking release is actually 5.2.4) causing problem with the test web host: https://github.com/aspnet/AspNetWebStack/issues/162.
Even better would be to change the test setup to use integrated pipeline, but I haven't found yet how to do that.